entity/src/organizations.rs

```diff
@@ -0,0 +1,32 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.12.3
+
+use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize)]
+#[sea_orm(schema_name = "refactor_platform", table_name = "organizations")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    #[serde(skip_deserializing)]
+    pub id: i32,
+    #[sea_orm(unique)]
+    pub external_id: Uuid,
+    pub name: Option<String>,
+    pub logo: Option<String>,
+    pub created_at: Option<DateTimeWithTimeZone>,
+    pub updated_at: Option<DateTimeWithTimeZone>,
```

Comment:
Same suggestion as for `Users` entity below, to make these non-optional fields defaulting to `now()` on creation.

Reply:
Updated in 360ce0c107cb7c45196f1823a05dfde432480c54!